DSI/globus_gridftp_server_iRODS.cpp

```diff
@@ -781,8 +780,11 @@ iRODS_connect_and_login(
         }
     }
 
+    printf("*****  iRODS: %s connected now logging in.\n", call_context_for_logging.c_str());
```

Comment:
Should this be removed?

Reply:
Good catch.  I put that in to debug the connection issue.  Will remove.